### ISSUE TITLE
A11y_Visual Studio Code Client_Output panel_keyboard:  Tooltip details for “Toggle search details ,Search only in open file and Use exclude settings" button, are not accessible using keyboard.

### ISSUE THREAD

**Issue by Shivanikupatkar** [microsoft/vscode#137208]
### GitHubTags

#A11yTCS; #A11ySev2; #A11yMAS; #A11yMAS #A11yWCAG; #DesktopApp; #Visual Studio Code Client;#BM-VisualStudioCode-Win32-Nov2021; #WCAG2.1.1; #Win32; #Keyboard;#Win10; #AINotINScope;

### Environment Details
Application Name: Visual Studio Code Client
Version:1.62.1 (user setup)
Windows Version: Windows 10 (21H1)

### Repro Steps
1) Open Visual studio code editor  
2) Press ctrl+shift+u.
3) Validate it's possible to toggle scroll lock.
4) Expand Search Details button using "Enter" key and  check when keyboard focus lands on  “Toggle search details ,Search only in open file and Use exclude settings" button, user is able to see the tooltips.

### Actual Result
Tooltip details for “Toggle search details ,Search only in open file and Use exclude settings" button, are not accessible using keyboard.

### Expected Result
Tooltip details for “Toggle search details ,Search only in open file and Use exclude settings" button, should be  accessible using keyboard.

### User Impact
Keyboard only users will have difficulty in accessing the Tooltips When keyboard focus lands on  “Toggle search details ,Search only in open file and Use exclude settings" button, user is not able to see the tooltips.

![A11y_Visual Studio Code Client_Sidebar Bar Options_keyboard](https://user-images.githubusercontent.com/94344363/141758716-b46489ed-b971-40bb-bf3d-bd64440d25b2.gif)

**Comment by roblourens** [comment]
I'm confused, you are referring to "Toggle search details" in the search view but not showing the search view, just the output pane? Is this about the search view or output pane?

**Comment by isidorn** [comment]
Dup of https://github.com/microsoft/vscode/issues/132344
